src/components/icon/icon.svelte

```diff
@@ -39,10 +51,14 @@
     /* Non color icons are set via a mask-image, so we can change the color
      * by setting the background */
     & .icon:not(.color) {
-      background: currentColor;
+      background: var(--icon-light-mode-color);
       -webkit-mask-image: var(--icon-url);
       mask-image: var(--icon-url);
       mask-repeat: no-repeat;
+
+      @media (prefers-color-scheme: dark) {
```

Comment:
This would need to be `@theme (dark)` to use our postcss plugin which accounts for dark mode "containers" provided by, e.g. `<div data-theme="dark">` wrappers for entire UI areas.